### DISCUSSION TITLE
What will u use?

### DISCUSSION THREAD

**Discussion by andmcpro** [community/community#147281] [Programming Help]
### Select Topic Area

Question

### Body

i have been learn something about web code, then i was notice what is worth to use in react application will u use **create react app** or **VITE** gimme youur opinions so i can choose waht most worthy

thx

**Comment by akadeepesh** [comment]
I will go with VITE

**Reply by andmcpro** [reply]
can u gimme some opinion, why u use vite?

**Reply by akadeepesh** [reply]
Yeah, for me it's easy to integrate typescript, build time is faster, it's lightweight...

**Comment by Logophoman** [comment]
Create-react-app is my go-to, it's easy and predictable and quite minimal, haven't used vite...

**Reply by andmcpro** [reply]
what is go-to?
im new so i didnt understand something like that

**Reply by Logophoman** [reply]
its the library i use for each and every react project, so i always go back to it 😉

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @andmcpro , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it to Programming Help for you. Good luck!

**Comment by GabrielMakeDrive** [comment]
Both create react app (CRA) and vite will get you up and running with react, but vite its more like you hit save and things update instantly, no long waits. Plus if youre into typescript or adding other features as you go, Vite makes it feel pretty painless. Dev experience is smoother overall

CRA still has its place with simplicity and familiarity. If youre new to react, CRA gives you a predictable, easy starting point, and there is a lot of beginner-friendly content out there to help you

**Comment by charmi-doshi** [comment]
As a web developer who has experience with both Create React App (CRA) and Vite, I believe the choice depends on what you're most comfortable with, as both have their pros and cons.

On a personal note, I’d recommend Vite because of its speed, scalability, automatic saves, and the fact that it's trending in modern web development. However, if you're a beginner, I suggest starting with CRA to get a solid foundation and then transitioning to Vite once you're more comfortable.

Hope this helps! 😊

**Comment by smit4k** [comment]
go with vite. fast, live updates, what more could you want
